cirq/circuits/circuit.py

```diff
@@ -1205,17 +1205,18 @@ def apply_unitary_effect_to_state(
         operations are present.
 
         For convenience, terminal measurements are automatically ignored
-        instead of causing a failure. Set the 'ignore_terminal_measurements'
+        instead of causing a failure. Set the `ignore_terminal_measurements`
```

Comment:
Note: in ReStructuredText, single backticks make it italic. Use two to make it code font. Since you say you found a way to use markdown in docstrings, maybe this does make it code font?